packages/design-picker/src/components/unified-design-picker.tsx

```diff
@@ -286,6 +284,12 @@ const DesignButtonContainer: React.FC< DesignButtonContainerProps > = ( {
 	const isDesktop = useViewportMatch( 'large' );
 	const isBlankCanvas = isBlankCanvasDesign( props.design );
 
+	if ( isBlankCanvas ) {
+		return isEnabled( 'signup/design-picker-pattern-assembler' ) ? (
```

Comment:
Do we need to check the flag? As without this flag, the blank canvas cannot be selected in the first place anyway.

Reply:
As we remove the blank canvas from the API before, we won't show it on the Design Picker currently. Consequently, we check the flag to avoid showing the blank canvas before we turn on the Pattern Assenbler

Reply:
Ah sorry, you are right!